pf/internal/check/check_test.go

```diff
@@ -31,9 +31,67 @@ import (
 
 	pfbridge "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
 	sdkv2 "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/sdk-v2"
 )
 
+type idSchema struct {
+	computed bool
+	required bool
+	optional bool
+}
+type mockSchema struct {
+	shim.Schema
+	idSchema
+}
+
+func (m *mockSchema) Optional() bool {
+	return m.optional
+}
+
+func (m *mockSchema) Required() bool {
+	return m.required
+}
+
+func (m *mockSchema) Computed() bool {
+	return m.computed
+}
+
+func TestIsInputProperty(t *testing.T) {
+	tests := []struct {
+		name    string
+		schema  shim.Schema
+		isInput bool
+	}{
+		{
+			name:    "Optional+Computed",
+			schema:  &mockSchema{idSchema: idSchema{optional: true, computed: true, required: false}},
+			isInput: true,
```

Comment:
To be pedantic I think this maps to both an input and an output property in Pulumi, but no changes needed here.